packages/docs-app/src/examples/core-examples/popoverExample.tsx

```diff
@@ -271,9 +271,13 @@ export class PopoverExample extends BaseExample<IPopoverExampleState> {
 
     private centerScroll = (div: HTMLDivElement) => {
         if (div != null) {
-            const container = div.parentElement;
-            container.scrollTop = div.clientHeight / 4;
-            container.scrollLeft = div.clientWidth / 4;
+            // if we don't requestAnimationFrame, this function apparently executes
+            // before styles are applied to the page, so the centering is way off.
```

Comment:
this sounds highly suspect... how are styles not on the page yet?

Reply:
Don't know, but IIRC, when I stopped at a breakpoint in here, nothing was styled on the page at all. @giladgray any idea why this might have happened?